### ISSUE TITLE
no userConfig.json to be found

### ISSUE THREAD

**Issue by chemolz9** [Soundnode/soundnode-app#1100]
While the Configuration section in the [README.md](https://github.com/Soundnode/soundnode-app/blob/master/README.md) states, that the ClientId is to be inserted in the file _userConfig.json_ under the path as detected in the [configLocation.js](https://github.com/eliecharra/soundnode-app/blob/master/app/public/js/common/configLocation.js#L34), there is no ${userHome}/.config directory in my Windows 8.1.
There is no userConfig.json to be found, neither under Users/..., nor in the soundnode appdata folder.

Is there any other way to store the clientId key in Soundnode, beside building it myself?

**Comment by kjbass** [comment]
I'm having the same experience here, cannot find the userConfig.json file. 
For reference I am on OSX Mojave.

**Comment by hbrs** [comment]
Same problem here. I'm on macOS Mojave

**Comment by jojojo8359** [comment]
Same problem, Ubuntu 18.04 LTS

**Comment by jakejarrett** [comment]
This was introduced in a later version, Please confirm you are using `7.0.0`, otherwise, you can download it from the releases page [here](https://github.com/Soundnode/soundnode-app/releases/tag/7.0.0)

**Comment by kjbass** [comment]
It's been quite a while since I was trying to get this to work, but, I'm 95% sure I was using 7.0.0, since I tried building from source.  Can try and reproduce the bug at some point, if it's only an "on my machine" sort of bug.

**Comment by jakejarrett** [comment]
Yeah if you're able to reproduce it on 7.0.0 then I can start working on implementing a fix for it. 

It definitely does appear locally on 7.0.1 for Ubuntu 18.04

**Comment by jojojo8359** [comment]
Ah, there lies the issue. I grabbed v0.6.5 from the website. Thanks for this clarification!

**Comment by AdityaPatange1** [comment]
I did a git clone of v7.0.0. How do I create the userConfig.json file? I can't find a copy anywhere.

**Comment by incognitoTheta** [comment]
Hi guys,

I am using the latest version 7.0.1-RC2 for Windows and I didn't find any userConfig.json file ( I used *.json in search bar for the main folder ). I am not a web developer so please tell me step by step what should I do to login with my soundcloud account with soundnode.

Thank you!
